README.md

```diff
@@ -2,23 +2,51 @@
 
 Hi, welcome to my project. 
 
-Due to using .gitignore, you will not be able to run this locally until you make some minor changes. 
+Summary of project: 
+(should evolve over time) This project is my first API.... 
 
-Please set up 2 .env files in the root directory, like this; 
+Link to project:
+https://chriss-new-app.onrender.com/api (you will find a list of all endpoints here)
+
+Instructions:
+Due to using .gitignore, you will not be able to run this locally or the hosted version until you make some minor changes. 
+
+Please set up 3 .env files in the root directory, like this; 
 
 File 1:
     File Name: 
         '.env.test'
     File Contents: 
-        'PGDATABASE=nc_news_test;'
+        PGDATABASE=nc_news_test;
 
 File 2:
     File Name: 
         '.env.development'
     File Contents: 
-        'PGDATABASE=nc_news;'
+        PGDATABASE=nc_news;
+
+File 3:
```

Comment:
I would remove this immediately, there are bots that scrape github for these tokens, generate a new password too as this will be in your git history permanently. The .env files are used to obfuscate private information.